src/main/webapp/i18n/de/multipleChoiceQuestion.json

```diff
@@ -23,7 +23,17 @@
                 "addCorrectAnswerOption": "[X] Correct Option",
                 "addIncorrectAnswerOption": "[ ] Incorrect Option",
                 "addHint": "[-h] Hint",
-                "addExplanation": "[-e] Explanation"
+                "addExplanation": "[-e] Explanation",
```

Comment:
This is NOT German! Please provide proper translations.
Would be great if you could translate the whole file actually ;)